tests/lib/pkgdb.sh

```diff
@@ -562,11 +558,22 @@ pkg_dependencies_ubuntu_classic(){
     esac
 }
 
+pkg_linux_image_extra(){
+    if apt-cache policy "linux-image-extra-$(uname -r)" >/dev/null 2>&1; then
```

Comment:
Thanks, this is much cleaner than the previous version. You could use `apt list "linux-image-extra-$(uname -r)"` here but this is fine.